// Copyright 2018 The ChromiumOS Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";
option optimize_for = LITE_RUNTIME;

package oobe_config;

// Data we want to save before rollback and restore after it happened.
message RollbackData {
  // Whether metrics are enabled.
  bool eula_send_statistics = 1;
  // Whether the EULA has already been accepted.
  bool eula_auto_accept = 2;
  // Network configuration in ONC format.
  string network_config = 3;
}
